features/logging/4.x/elasticsearch.feature

```diff
@@ -0,0 +1,65 @@
+@clusterlogging
+Feature: Elasticsearch related tests
+
+  # @auther qitang@redhat.com
+  # @case_id OCP-22050
+  @admin
+  @destructive
+  Scenario: Elasticsearch using dynamic volumes
+    Given I switch to cluster admin pseudo user
+    And default storageclass is stored in the :default_sc clipboard
+    Given I delete the clusterlogging instance
+    Then the step should succeed
+    And I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/example_unmanaged.yaml |
+    Then the step should succeed
+    Given I use the "openshift-logging" project
+    Given I wait for the "instance" clusterloggings to appear
+    Given I get project clusterlogging named "instance" as YAML
+    Then the step should succeed
+    And the output should contain:
+      | managementState: Unmanaged |
+    Given I get project elasticsearch named "elasticsearch" as YAML
+    Then the step should fail
+    When I run the :patch client command with:
+      | resource      | clusterlogging                                                                                        |
+      | resource_name | instance                                                                                              |
+      | p             | {"spec":{"logStore":{"elasticsearch":{"storage":{"storageClassName": "<%= cb.default_sc.name %>"}}}}} |
+      | type          | merge                                                                                                 |
+    Then the step should succeed
+    Given I get project clusterlogging named "instance" as YAML
+    And the output should contain:
+      | storageClassName: <%= cb.default_sc.name %> |
+      | managementState: Unmanaged                  |
+    Given I get project elasticsearch named "elasticsearch" as YAML
+    Then the step should fail
+    When I run the :patch client command with:
+      | resource      | clusterlogging                         |
+      | resource_name | instance                               |
+      | p             | {"spec":{"managementState":"Managed"}} |
+      | type          | merge                                  |
+    Then the step should succeed
+    Given I get project clusterlogging named "instance" as YAML
+    And the output should contain:
+      | storageClassName: <%= cb.default_sc.name %>   |
+      | managementState: Managed                      |
+    Given I wait for the "elasticsearch" elasticsearches to appear
+    Given I get project elasticsearch named "elasticsearch" as YAML
```

Comment:
maybe create a class here too?

Reply:
I will not block merging for these classes. But if @pruan-rht wants to help, that would be arwesome.

Reply:
@QiaolingTang @akostadinov +1 on writing a class interface for those two new resources.  I'll work on them Monday or perhaps tomorrow if I can find some time.

Reply:
@pruan-rht @akostadinov I've add some classes, but I forget to update the case. https://github.com/openshift/verification-tests/pull/379/files#diff-b7b17bd879fb9cd97bf091211d2e34ea and https://github.com/openshift/verification-tests/pull/379/files#diff-bc2107c36e302368fe4b4afe755439de

Reply:
I've updated several classes in this PR. But for some fields in the elasticsearch CRD instance, I haven't figured out a good way to process them, so I keep using step `I get project elasticsearch named "elasticsearch" as YAML`.